website/client/src/components/auth/registerLoginReset.vue

```diff
@@ -777,6 +777,7 @@ export default {
       // windows: WINDOWS_CLIENT_ID,
       google: process.env.GOOGLE_CLIENT_ID, // eslint-disable-line
     });
+    this.setTitle();
```

Comment:
I think this might not be necessary beacuse the watched on $route has `immediate: true` so it'll fire right away

Reply:
Yes I agree